settings/LoanPolicyForm.js

```diff
@@ -164,6 +170,17 @@ class LoanPolicyForm extends React.Component {
         </FormattedMessage>
       ));
 
+    const setTimeOffsetVisibility = () => {
```

Comment:
Seems like the name of the function does not fully reflect the meaning of the function. The function itself does not `set` anything. Rather it returns something. Should it start with `get`?